config-model-api/src/main/java/com/yahoo/config/application/api/ApplicationPackage.java

```diff
@@ -71,9 +72,6 @@ public interface ApplicationPackage {
     String RULES_NAME_SUFFIX = ".sr";
     String EXT_DIR = "ext";
 
-    // TODO: Remove when oldest model version is 8.95
```

Comment:
Needed until Vespa 7 is gone?

Reply:
I don't think so, but since I'm not 100% sure I put it back.